presto-tests/src/test/java/com/facebook/presto/tests/TestLocalBinarySpilledQueries.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.tests;
-
-import com.facebook.presto.Session;
-import com.facebook.presto.SystemSessionProperties;
-import com.facebook.presto.connector.ConnectorId;
-import com.facebook.presto.metadata.SessionPropertyManager;
-import com.facebook.presto.sql.analyzer.FeaturesConfig;
-import com.facebook.presto.testing.LocalQueryRunner;
-import com.facebook.presto.tpch.TpchConnectorFactory;
-import com.google.common.collect.ImmutableMap;
-
-import java.nio.file.Paths;
-
-import static com.facebook.presto.testing.TestingSession.TESTING_CATALOG;
-import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
-import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
-
-public class TestLocalBinarySpilledQueries
-        extends AbstractTestQueries
-{
-    public TestLocalBinarySpilledQueries()
-    {
-        super(TestLocalBinarySpilledQueries::createLocalQueryRunner);
-    }
-
-    private static LocalQueryRunner createLocalQueryRunner()
```

Comment:
(cmt msg)
> In case of failure it is easy to change the query runner to see if given tests are passing with local query executor.

This looks like over-advertised. Judging by the amount of red here, it's not that easy.